cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,126 @@
 /// <reference types='cypress' />
+let worker
 
 describe('Web Tables page', () => {
-  it('', () => {
 
+  beforeEach(() => {
+    cy.task('newWorker').then(newWorker => {
+      worker = newWorker;
+   });
+    cy.visit('/');
+    
   });
+   
+  it('The page should contain the pagination', () => {
+
+    cy.get('.-pageInfo').should('contain.text', 'Page  of 1');
+    cy.contains('[disabled]', 'Previous').should('contain', 'Previous');
+    cy.contains('[disabled]', 'Next').should('contain', 'Next');
+  });
+
+  it('Should provide an ability to select rows count', () => {
+    cy.get('select').select('5 rows').should('have.value', '5');
+
+   cy.get('select').select('10 rows').should('have.value', '10');
+
+  
+   cy.get('select').select('25 rows').should('have.value', '25')
+
+  });
+
+  it('Should provide an ability to add new worker', () => {
+
+
+   cy.contains('#addNewRecordButton', 'Add').click();
+   cy.findByPlaceholder('First Name').type(worker.firstName);
+   cy.findByPlaceholder('Last Name').type(worker.lastName);
+   cy.findByPlaceholder('name@example.com').type(worker.email);
+   cy.findByPlaceholder('Age').type(worker.age);
+   cy.findByPlaceholder('Salary').type(worker.salary);
+   cy.findByPlaceholder('Department').type(worker.department);
+   cy.contains('#submit', 'Submit').click();
+   cy.contains('.rt-tr', worker.firstName).should('contain', worker.firstName);
+   cy.contains('.rt-tr', worker.lastName).should('contain', worker.lastName);
+   cy.contains('.rt-tr', worker.age).should('contain', worker.age);
+   cy.contains('.rt-tr', worker.salary).should('contain', worker.salary);
+   cy.contains('.rt-tr', worker.department).should('contain', worker.department);
+
+  })
+
+  it('Should provide an ability to delete worker', () => {
+    cy.contains('#addNewRecordButton', 'Add').click();
+    cy.findByPlaceholder('First Name').type(worker.firstName);
+    cy.findByPlaceholder('Last Name').type(worker.lastName);
+    cy.findByPlaceholder('name@example.com').type(worker.email);
+    cy.findByPlaceholder('Age').type(worker.age);
+    cy.findByPlaceholder('Salary').type(worker.salary);
+    cy.findByPlaceholder('Department').type(worker.department);
+    cy.contains('#submit', 'Submit').click();
+
+    cy.get('#delete-record-4').click();
+    cy.contains('.rt-tr', worker.department).should('not.be.an', worker.department);
```

Comment:
![image](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/9807f0dd-a61f-45d6-9cb1-cfde4d1d3c33)
you have strange assertions here) Fix this test and check that the table does not contain a deleted user name and surname.